addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -276,6 +278,104 @@ odoo.define('website_slides.fullscreen', function (require) {
         },
     });
 
+    var ShareDialog = Dialog.extend({
+        template: 'website.slide.share.modal',
+        events: {
+            'click .oe_slide_js_share_email button': '_onShareByEmailClick',
```

Comment:
Small guidelines remark: it is better to keep the standard prefixing o_wslides_something. Wslides is not pretty but it's standard :p .